# Exception: Description, Traceback
# for i in range(5)
#     print("Hello, world!")

# print(1 / 0)

# with open("./x_file.txt") as xf:
#     the_truth = xf.read()
# print(the_truth)

# print(1 + 2 + "three")

import math  # cmath module

print(math.sqrt(-1))
